app/src/main/java/com/example/mayankjindal/voicecalculator/MainActivity.java

```diff
@@ -28,25 +25,20 @@
 import java.util.ArrayList;
 
 public class MainActivity extends AppCompatActivity {
-    // IDs of all the numeric buttons
     private int[] numericButtons = {R.id.btnZero, R.id.btnOne, R.id.btnTwo, R.id.btnThree, R.id.btnFour, R.id.btnFive, R.id.btnSix, R.id.btnSeven, R.id.btnEight, R.id.btnNine};
-    // IDs of all the operator buttons
     private int[] operatorButtons = {R.id.btnAdd, R.id.btnSubtract, R.id.btnMultiply, R.id.btnDivide};
-    // TextView used to display the output
-    private TextView txtScreen;
-    // Represent whether the lastly pressed key is numeric or not
-    private boolean lastNumeric;
+    private TextView txtScreen; //Display Output
+    private boolean lastNumeric; //Lastly Pressed Key is Numeric or not
     // Represent that current state is in error or not
     private boolean stateError;
     // If true, do not allow to add another DOT
     private boolean lastDot;
 
     private static final int REQUEST_CODE = 1234;
-    Button Start;
-    TextView Speech;
-    Dialog match_text_dialog;
-    ListView textlist;
-    ArrayList<String> matches_text;
+    Button start;
+    Dialog matchTextDialog;
```

Comment:
@rishabhdeepsingh Good work. You have followed the java convention for naming of variables.